contentctl/objects/abstract_security_content_objects/detection_abstract.py

```diff
@@ -51,8 +51,8 @@
 # TODO (#266): disable the use_enum_values configuration
 class Detection_Abstract(SecurityContentObject):
     model_config = ConfigDict(use_enum_values=True)
-
-    # contentType: SecurityContentType = SecurityContentType.detections
+    name:str = Field(...,max_length=67)
```

Comment:
Should this magic number, which comes from the max length of a stanza after editing being updated to 99, be defined somewhere?

Reply:
No more mysterious constants or strange magic numbers - these are now determined from some constant limits (with documentation) and calculated at runtime using some string templates: https://github.com/splunk/contentctl/pull/179/files#diff-1fd20d9a216cf9dcfd3a44fdd0ef12fc5f04246124b5b5eb8aa85c7706b512ecR164-R172 

@ljstella , please resolve if this looks good to you!